src/weierstrass/curve.rs

```diff
@@ -187,41 +199,43 @@ const ERR_ECC_NEW: &str = "not valid point";
 // tests for ser/deser
 #[cfg(test)]
 mod tests {
-    use crate::instances::{P256, P384, P521, GetCurve};
-    use crate::ellipticcurve::{EllipticCurve, Encode, Decode};
+    use crate::ellipticcurve::{Decode, EllipticCurve, Encode};
     use crate::field::Field;
+    use crate::instances::{GetCurve, P256, P384, P521};
 
     #[test]
     fn point_serialization() {
-        for &id in [P256,P384,P521].iter() {
+        for &id in [P256, P384, P521].iter() {
             let ec = id.get();
             let modulus = ec.get_field().get_modulus();
             let gen = ec.get_generator();
             let ser = gen.encode(false);
-            assert_eq!(ser.len(), 2*((modulus.bits()+7)/8)+1);
+            assert_eq!(ser.len(), 2 * ((modulus.bits() + 7) / 8) + 1);
```

Comment:
It may be worth catching the length of bytes required to express the modulus in a function for each curve, as we do the same operations in a number of places?